.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Draft release
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write # For creating releases.
+  discussions: write #  For creating a discussion.
+
+# Cancel in-progress runs for pull requests when developers push
+# additional changes
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Download artifacts
+        uses: dawidd6/action-download-artifact@v2
+        id: download
+        with:
+          branch: main
```

Comment:
Actually do we need the branch?  We could just use whatever artifacts are on the branch we set the workflow to target when dispatching it I think?  That would let us release off different branches too if for some reason we needed to do that (like if we needed to backport a fix).

Reply:
I guess not! That's a good point so I guess I'll change this to `${{ github.ref }}`